doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -543,6 +543,9 @@ Libraries for networking
 
   * Fixed the warning due to missing ``https`` download protocol.
 
+* Updated the :ref:`lib_downloader` library to support Proxy-URI option and an authentication callback after connecting.
+* Updated the :ref:`lib_fota_download` and :ref:`lib_nrf_cloud_fota` libraries to use the :ref:`lib_downloader` library for CoAP downloads.
```

Comment:
```suggestion
* :ref:`lib_downloader` library:

  * Updated to support Proxy-URI option and an authentication callback after connecting.

* :ref:`lib_fota_download` library:

  * Updated to use the :ref:`lib_downloader` library for CoAP downloads.

* :ref:`lib_nrf_cloud_fota` library:

  * Updated to use the :ref:`lib_downloader` library for CoAP downloads.
```

Reply:
Are you referring to the lib_nrf_cloud (https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/libraries/networking/nrf_cloud.html) library here? lib_nrf_cloud_fota (Firmware over-the-air (FOTA) updates) is a section in this library.

Reply:
fixed